core/python/test/optimizers.py

```diff
@@ -0,0 +1,102 @@
+import os
```

Comment:
Remove this file when the c++ version of this test is implemented.
It is not necessary to do the test both in c++ and python - since this test does not intend to test the Python Bindings, it should not be written in Python.